### PR TITLE
DOC-4038

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

File: modules/ROOT/pages/managing-connections.adoc [hunk 1]
```diff
@@ -49,6 +49,21 @@ var bucket = cluster.openBucket('protectedBucket', function(err) {
 });
 ----
 
+[NOTE]
+====
+
+From Couchbase Node.js SDK 2.5.0 (using _libcouchbase_ 2.9.2), [.api]`AlternateAddress` is implemented, for connecting to nodes in a NATed environment, such as Docker containers using portmapping.
```
Comment (brett19) [inline modules/ROOT/pages/managing-connections.adoc:55]: You use 'AlternateAddress' here, but then call it Multi-Network Configuration below.  Why is this?

File: modules/ROOT/pages/managing-connections.adoc [hunk 2]
```diff
@@ -49,6 +49,21 @@ var bucket = cluster.openBucket('protectedBucket', function(err) {
...
+It is on by default, if the server provides a topology that includes a multi-network configuration.
+Whichever network is selected at bootstrap will be logged.
+
+If using __Docker Swarm__, or otherwise running the SDK inside the NAT, then you will want to disable with `?network = internal` in the connection string, or an xref:client-settings.adoc[environmental setting] can be made.
```
Comment (brett19) [inline modules/ROOT/pages/managing-connections.adoc]: This is going to be discussed a bit more.  I think there is an error in our RFC.

Comment (RichardSmedley) [reply modules/ROOT/pages/managing-connections.adoc]: What can I put through now that reflects the implementation in Node.js right now?
There's some demand for docs on this ;)

File: modules/ROOT/pages/managing-connections.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/ROOT/pages/managing-connections.adoc:55]: I'm going with AlternateAddress as the implementation of multi network configuration, 
we did something similar in the RTO / Tracing docs - what would be a better alternative?

File: modules/ROOT/pages/managing-connections.adoc (hunk 2 above)
Comment (brett19) [reply modules/ROOT/pages/managing-connections.adoc]: Looks like 'internal' should really say 'default'.  That was my error.  There should be no space between there as well.

**Review by brett19 — APPROVED** [review]

**Pull request merged**